src/formatDate.js

```diff
@@ -50,7 +50,66 @@
  */
 
 function formatDate(date, fromFormat, toFormat) {
-  // write code here
+  const dateArr = date.split(fromFormat[3]);
+  const [year, fromYear, toYear, indexOfYearFrom]
+    = findYears(dateArr, fromFormat, toFormat);
+
+  if (fromYear.length > toYear.length) {
+    dateArr[indexOfYearFrom] = year.toString().slice(2);
+  }
+
+  if (fromYear.length < toYear.length && year < 30) {
+    dateArr[indexOfYearFrom] = '20' + year;
+  }
+
+  if (fromYear.length < toYear.length && year >= 30) {
+    dateArr[indexOfYearFrom] = '19' + year;
+  }
```

Comment:
use ternary operator for the condition `fromYear.length < toYear.length && year >= 30`
https://developer.mozilla.org/en-US/docs/Web/JavaScript/Reference/Operators/Conditional_Operator